### ISSUE TITLE
Create possibility to generate "readable links"

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1147]
In connection with the new edition of the OER-Atlas it would be useful to have human readable links (sprechende Links) for the entries which we could put in the print version.

In an ideal world I have the possibility from the entry to click on “Create readable link” and than have the possibility to customize it. 

Would this be possible? Would it be very time consuming? Technically difficult?

**Comment by acka47** [comment]
Are you talking about something like `https://oerworldmap.org/resource/hbz` for the hbz? I don't think we should do this because some problems arise:

- Based on what field a "readable" link should be created?
- What happens when the content of the field is changed? Do we have to support two different "readable" links?
- Most important: How do we guarantee that these links will work in the future?

An OER World Map link shortener would probably be a better option for adding links to print publications. In the past, we have already thought about setting up a hbz-wide link shortener, see https://github.com/hbz/lobid/issues/278.

**Comment by karindr** [comment]
I thought of something like the service of bitly: The system suggests a short link and you can customize it.

> Based on what field a "readable" link should be created?

This is done by the human creator, so there is no problem.

> What happens when the content of the field is changed? Do we have to support two different "readable" links?

The original url of the entry is staying the same. If the name changes the person could create a second shortened url.

> Most important: How do we guarantee that these links will work in the future?

If the original url is not changing, why is this a problem?

> An OER World Map link shortener would probably be a better option for adding links to print publications. In the past, we have already thought about setting up a hbz-wide link shortener, see hbz/lobid#278.

This would be the second best solution.

**Comment by acka47** [comment]
Obviously, we are very close together in this. If I understand you correctly, you want a URL shortener that has the possibility to create custom URLs. For example, both http://techno-weenie.net/guillotine/ and https://github.com/YOURLS/YOURLS are capable of this.

**Comment by literarymachine** [comment]
Use external services for this.

**Comment by programmieraffe** [comment]
I would like to put this back on the agenda.

I would be totally okay with https://oerworldmap.org/resource/urn:uuid:839897e6-178f-11e5-871b-3c970e4a9cbf/jan-neumann/

or something like that, but the current approach is just not good quality because it can't be read by humans. :/
